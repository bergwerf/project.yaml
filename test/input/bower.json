{
  "name": "YourAwesomePackageName",
  "main": "dist/your-awesome-js-module.js",
  "ignore": [
    "source",
    "spec",
    ".bowerrc",
    ".gitignore",
    ".jshintignore",
    ".jshintrc",
    "bower.json",
    "gruntfile.js",
    "package.json",
    "README.md"
  ],
  "dependencies": {
    "jquery": ">=1.10.2",
    "underscore": "~1.5.2"
  },
  "devDependencies": {
    "jquery-mockjax": "~1.5.2"
  }
}
