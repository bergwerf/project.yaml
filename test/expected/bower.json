{
    "name": "My Project",
    "main": "dist/your-awesome-js-module.js",
    "ignore": [
        "source",
        "spec",
        ".bowerrc",
        ".gitignore",
        ".jshintignore",
        ".jshintrc",
        "bower.json",
        "gruntfile.js",
        "package.json",
        "README.md"
    ],
    "dependencies": {
        "jquery": ">=1.10.2",
        "underscore": "~1.5.2"
    },
    "devDependencies": {
        "jquery-mockjax": "~1.5.2"
    },
    "authors": [
        "John Doe",
        "Joe Bloggs",
        "Joe Bloggs <joe@bloggs.com>",
        "Joe Bloggs <joe@bloggs.com> (http://joebloggs.com)"
    ],
    "description": "Awesome project!",
    "license": "AGPL-3.0",
    "homepage": "http://example.com",
    "repository": {
        "type": "git",
        "url": "https://github.com/username/repository.git"
    }
}
